### DISCUSSION TITLE
[Cheat] Stop Time

### DISCUSSION THREAD

**Discussion by SuperBro64** [HarbourMasters/2ship2harkinian#673] [Ideas]
Although it is currently possible to stop time from passing by going into the Save Editor, I think an instant cheat for doing the same like in SoH would be a bit more convenient.

**Comment by Retroboy85** [comment]
I had just posted about adding this, I had created these exact cheat codes for this.

you could manually set which day you are on and also Fast-forward/Re-wind the flow of time in real-time (like you can watching a film).

I posted my cheat codes over at GameHacking.org for the PAL region, but I can easily port them over to other regions.

I asked if they wanted to also include my cheat code because I have to keep manually adding it into PJ64 as I am setting up on my ONEXPLAYER 2 PRO, so its not in their master cheat list.

But I am sure users using this would like it if they can use the cheats I made.
https://gamehacking.org/game/20551?hacker=Retrojunkies

**Comment by Retroboy85** [comment]
This is an example of my Control Day/Time cheat code for Legend of Zelda, The - Majora's Mask (Europe) (En,Fr,De,Es)

D03DDFDB 0011
801E6B66 0001
D03DDFDB 0000
801E6B66 0000
D03DDFDB 0012
811E6B66 FFAA
D03DDFDB 0000
811E6B66 0000
D03DDFDB 0038
801E6B68 0002
D03DDFDB 0034
801E6B68 0000
D03DDFDB 0028
801E6B6B 0001
D03DDFDB 0022
801E6B6B 0002
D03DDFDB 0021
801E6B6B 0003

R+C > =Forward Time / R+C < =Rewind Time
L+R+C ^ =Remove Time / L+R+C V =Display Time 
L+C ^ =Day One
L+C < =Day Two
L+C > =Day Three

If you're not wanting the cheat code as an AIO, I've added each address below with what it does.

Forward Time
D03DDFDB 00XX < XX = Button Activator Pressed 
801E6B66 0001 < Speeds Time Forward
D03DDFDB 0000 < If No Buttons Pressed
801E6B66 0000 < Returns Flow of Time to Normal

Rewind Time
D03DDFDB 00XX < XX = Button Activator Pressed
811E6B66 FFAA < Rewinds Time Backwards
D03DDFDB 0000 < If No Button Pressed
811E6B66 0000 < Returns Flow of Time to Normal

Hide/Display Time
D03DDFDB 00XX < XX = Button Activator Pressed
801E6B68 0002 < Hides Time
D03DDFDB 00XX < XX = Button Activator Pressed
801E6B68 0000 < Displays Time

Day X
D03DDFDB 00XX < XX = Button Activator Pressed
801E6B6B 000X < Values 1 - 3 to set Day ( Value 4 = Dawn of a New Day - removes all events)

I could port to other regions if needed.
